File: app/src/main/java/za/co/gundula/app/arereyeng/ui/AReYengCardsActivity.java
package za.co.gundula.app.arereyeng.ui;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import butterknife.BindView;
import butterknife.ButterKnife;
import za.co.gundula.app.arereyeng.R;
import za.co.gundula.app.arereyeng.model.Agency;

import static za.co.gundula.app.arereyeng.utils.Constants.agency_key;

public class AReYengCardsActivity extends AppCompatActivity {

    @BindView(R.id.toolbar)
    Toolbar toolbar;

    Agency agency = null;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_are_yeng_cards);

        ButterKnife.bind(this);

        Bundle bundle = getIntent().getExtras();
        if (bundle != null) {
            agency = bundle.getParcelable(agency_key);
        }

        toolbar.setTitle(getResources().getString(R.string.areyeng_cards));
        setSupportActionBar(toolbar);

        //noinspection ConstantConditions
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

    }
}

File: app/src/main/java/za/co/gundula/app/arereyeng/ui/AReYengCardsActivity.kt
package za.co.gundula.app.arereyeng.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import butterknife.BindView
import butterknife.ButterKnife
import za.co.gundula.app.arereyeng.R
import za.co.gundula.app.arereyeng.model.Agency
import za.co.gundula.app.arereyeng.utils.Constants

class AReYengCardsActivity : AppCompatActivity() {

    @BindView(R.id.toolbar)

    var toolbar: Toolbar? = null
    var agency: Agency? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_are_yeng_cards)
        ButterKnife.bind(this)
        val bundle = intent.extras
        if (bundle != null) {
            agency = bundle.getParcelable(Constants.agency_key)
        }
        toolbar!!.title = resources.getString(R.string.areyeng_cards)
        setSupportActionBar(toolbar)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
    }
}